Common/tests/Auth/AuthenticationIntegrationTest.cs

```diff
@@ -110,40 +111,43 @@ await helper_.StopServer()
+  private static readonly Session                                        SessionRequest;
+  private static readonly GetResultStatusRequest                         GetResultStatusRequest;
+  private static readonly GetTaskStatusRequest                           GetTaskStatusRequest;
+  private static readonly Empty                                          Empty;
+  private static readonly SessionFilter                                  SessionFilter;
+  private static readonly ResultRequest                                  ResultRequest;
+  private static readonly TaskOutputRequest                              TaskOutputRequest;
+  private static readonly WaitRequest                                    WaitRequest;
+  private static readonly CreateLargeTaskRequest                         CreateLargeTaskRequestInit;
+  private static readonly CreateLargeTaskRequest                         CreateLargeTaskRequestInitTask;
+  private static readonly CreateLargeTaskRequest                         CreateLargeTaskRequestPayload;
+  private static readonly CreateLargeTaskRequest                         CreateLargeTaskRequestPayloadComplete;
+  private static readonly CreateLargeTaskRequest                         CreateLargeTaskRequestLastTask;
+  private static readonly CancelSessionRequest                           CancelSessionRequest;
+  private static readonly GetSessionRequest                              GetSessionRequest;
+  private static readonly ListSessionsRequest                            ListSessionsRequest;
+  private static readonly GetResultIdsRequest                            GetResultIdsRequest;
+  private static readonly GetTaskRequest                                 GetTaskRequest;
+  private static readonly ListTasksRequest                               ListTasksRequest;
+  private static readonly GetOwnerTaskIdRequest                          GetOwnerTaskIdRequest;
+  private static readonly ListApplicationsRequest                        ListApplicationsRequest;
+  private static readonly CancelTasksRequest                             CancelTasksRequest;
+  private static readonly ListResultsRequest                             ListResultsRequest;
+  private static readonly GetCurrentUserRequest                          GetCurrentUserRequest;
+  private static readonly EventSubscriptionRequest                       EventSubscriptionRequest;
+  private static readonly ListPartitionsRequest                          ListPartitionsRequest;
+  private static readonly GetPartitionRequest                            GetPartitionRequest;
+  private static readonly CountTasksByStatusRequest                      CountTasksByStatusRequestApplications;
+  private static readonly Api.gRPC.V1.Sessions.CountTasksByStatusRequest CountTasksByStatusRequestSessions;
+  private static readonly Api.gRPC.V1.Tasks.CountTasksByStatusRequest    CountTasksByStatusRequestTasks;
```

Comment:
Couldn't you use a namespace alias to avoid the really long FQN?

Reply:
I use it for the first application. The namespace proposition is not really great

Reply:
No, I mean something like:
```c#
using grpc = Api.gRPC.V1;
```

In retrospective, I am not sure this would gain much.

Reply:
> I use it for the first count..... The namespace proposition is not really great

You should avoid using the non qualified name for the Application one to make it consistent, though.

Reply:
I'm not really convinced for this one.
Iwould have prefered something like

```csharp
using ApplicationCountTasksByStatusRequest Api.gRPC.V1.Applications.CountTasksByStatusRequest
```

Reply:
> > I use it for the first count..... The namespace proposition is not really great
> 
> You should avoid using the non qualified name for the Application one to make it consistent, though.

well, jetbrains is not ok with that

Reply:
That's a bummer...